packages/qowaiv/src/Clock.ts

```diff
@@ -0,0 +1,22 @@
+import { DateOnly } from "./DateOnly";
+
```

Comment:
The files look technicaly fine. 

Keep in mind this is build with .net approach. Npm packages usually have a more functional approach.
Which is broader accepted in the community. 

.net orientated developers will easily use this non .net less so.

This file can be written 

export const clock = {
  generator: () => new Date(),

  now(): Date {
    return clock.generator();
  },

  today(): DateOnly {
    const now = clock.now();
    return new DateOnly(now.getFullYear(), now.getMonth(), now.getDate());
  }
}

Or even as 

export const now = () => new Date();
export const today = () => {
  const now = new Date();
  return new DateOnly(now.getFullYear(), now.getMonth(), now.getDate());
};